__tests__/feeds.ts

```diff
@@ -1196,8 +1196,8 @@ describe('query authorFeed', () => {
 });
 
 describe('query mostUpvotedFeed', () => {
-  const QUERY = (period = 7, first = 10): string => `{
-    mostUpvotedFeed(first: ${first}, period: ${period}) {
+  const QUERY = (period = 7, first = 10, source = '', tag = ''): string => `{
+    mostUpvotedFeed(first: ${first}, period: ${period}, source: "${source}", tag: "${tag}") {
```

Comment:
Just to double check existing frontend queries won't break due to missing tag/source in their queries?

Reply:
![Screenshot 2024-04-23 at 13 33 42](https://github.com/dailydotdev/daily-api/assets/554874/512eca37-4d51-46c8-ac8a-fd6956d9b7a4)

Yeah works, if it's not set it's just not queried.